nuget/nunitlite/nunitlite.snupkg.nuspec

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata minClientVersion="3.5">
+    <id>NUnitLite</id>
+    <title>NUnitLite</title>
+    <version>$version$</version>
+    <packageTypes>
+      <packageType name="SymbolsPackage" />
+    </packageTypes>
+    <authors>Charlie Poole, Rob Prouse</authors>
+    <owners>Charlie Poole, Rob Prouse</owners>
+    <license type="file">LICENSE.txt</license>
+    <projectUrl>https://nunit.org</projectUrl>
+    <repository type="git" url="https://github.com/nunit/nunit"/>
+    <iconUrl>https://cdn.rawgit.com/nunit/resources/master/images/icon/nunit_256.png</iconUrl>
+    <icon>icon.png</icon>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Symbols for debugging into the NUnitLite assemblies.</description>
+    <copyright>Copyright (c) 2021 Charlie Poole, Rob Prouse</copyright>
+  </metadata>
+  <files>
+    <file src="..\..\nuget\icon.png" />
+    <file src="bin/net35/nunitlite.pdb" target="lib\net35" />
```

Comment:
Do we also need a `<file src="LICENSE.txt" />` or is this implicit from `<license type="file">LICENSE.txt</license>` ?

Reply:
You can't include anything but PDBs in the `snupkg`, so license and the icon aren't included and not working. I removed the icon and changed the license to `<license type="expression">MIT</license>`. I don't think much of this matters since these packages are not displayed.